source/aggregation.txt

```diff
@@ -82,20 +81,46 @@ Consider the following limitations when performing aggregation operations:
      </reference/operator/aggregation/graphLookup/>` stage has a strict
      memory limit of 100 megabytes and ignores the ``allowDiskUse`` option.
 
-.. _php-aggregation-example:
+Aggregation APIs
+----------------
 
-Aggregation Example
--------------------
+The {+library-short+} provides the following APIs to create aggregation
+pipelines:
+
+- :ref:`php-aggregation-array-api`: Create aggregation pipelines by
+  passing arrays that specify the aggregation operators and parameters 
+- :ref:`php-aggregation-builder-api`: Create aggregation pipelines by using native
```

Comment:
Q: Any guidelines on linking to sections on the same page within text and without a proper introduction? Was a little thrown off when this just led me to the section right beneath. If the team usually does this, it's alright. Could remove links if not.